app/scripts/metamask-controller.js

```diff
@@ -4913,10 +4940,6 @@ export default class MetamaskController extends EventEmitter {
             this.networkController,
           ),
         setActiveNetwork: (networkClientId) => {
-          // setActiveNetwork is used for custom (non-infura) networks, and for these networks networkClientId === networkConfigurationId
-          this.selectedNetworkController.setNetworkClientIdForMetamask(
-            networkClientId,
-          );
```

Comment:
this is no longer necessary because `SelectedNetworkController` no longer keeps track of the globally `selectedNetworkClientId`